gen3/bin/dcf.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+AWS_INPUT_BUCKET="data-refresh-manifest"
+AWS_OUTPUT_BUCKET="data-refresh-output"
+GS_INPUT_BUCKET="replication-input"
+GS_OUTPUT_BUCKET="datarefresh-log"
+
+echo "Hello from dcf!"
+command=$1
+release=$2
+
+# Activate google service account
+if [ -f "${WORKSPACE}/${vpc_name}/apis_configs/dcf_dataservice/creds.json" ]; then
```

Comment:
Use the `g3auto` folder for secrets.  Also - if it's not too much trouble - use a different name than `creds.json`, since we already use that for database creds for other services - maybe `googleCreds.json` - something like that ex:
```
$(gen3_secrets_folder)/g3auto/dcf_dataservice/googleCreds.json
```

Reply:
I made the change in d94ef65a821d039b329d3e21533b9a9d809fef7c
Thanks!